PhysicsTools/JetMCAlgos/python/AK4GenJetFlavourInfos_cfi.py

```diff
@@ -1,5 +1,11 @@
 import FWCore.ParameterSet.Config as cms
 from PhysicsTools.JetMCAlgos.AK4PFJetsMCFlavourInfos_cfi import ak4JetFlavourInfos
 
-ak4GenJetFlavourInfos = ak4JetFlavourInfos.clone(jets = "ak4GenJetsNoNu")
+ak4GenJetFlavourInfos = ak4JetFlavourInfos.clone(
+											jets = "ak4GenJetsNoNu",
+											bHadrons = cms.InputTag("selectedHadronsAndPartonsForGenJetsFlavourInfos","bHadrons"),
+    										cHadrons = cms.InputTag("selectedHadronsAndPartonsForGenJetsFlavourInfos","cHadrons"),
+  											partons  = cms.InputTag("selectedHadronsAndPartonsForGenJetsFlavourInfos","physicsPartons"),
+    										leptons  = cms.InputTag("selectedHadronsAndPartonsForGenJetsFlavourInfos","leptons"),
```

Comment:
please fix the indentation.
It looks like the problem is with the way how tabs are displayed: for this just replace tabs with spaces